frontend/app/assets/javascripts/spree/frontend/views/spree/shared/nav_bar.js

```diff
@@ -1,4 +1,4 @@
-Spree.ready(function() {
+$(document).ready(function() {
```

Comment:
@PSzyszkaSpark shouldn't this be `document.addEventListener('turbolinks:load', function () {` ?

Reply:
I will try it out.

Reply:
@damianlegawiec If we add `document.addEventListener('turbolinks:load', function () {` it behaves the same way as `Spree.ready(function() {` and it enters multiple times in `searchIcons.addEventListener` what we want to avoid.
For example if it enters 2, 4, 6.. times it will add class `shown` and after it it will be removed.